src/app/Shopping/Detail/page.tsx

```
@@ -0,0 +1,162 @@
+import styled from "styled-components/native";
+import { color, Font } from "../../../styles"
+import { Header, Footer } from "../../../components/Main";
+import { Tab, ProductCardSmall, Tag } from "../../../components/Shopping";
```

ESLint

[error] 'ProductCardSmall' is defined but never used.

(@typescript-eslint/no-unused-vars)